### ISSUE TITLE
rgbt234 dataset raw result

### ISSUE THREAD

**Issue by ShangGaoG** [RyanHTR/TBSI#3]
Thank you for your great contribution to the community! Could you provide the raw result on RGB234?

**Comment by Zhangyong-Tang** [comment]
Have you reproduced tbsi on rgbt234?
I tried to reproduce the officially published model and retrain the network, but the results are both 0.613/0.842, with a large gap between the results listed in the paper.

**Comment by RyanHTR** [comment]
Thanks for your comment. The raw results of rgbt234 and rgbt210 are on the dev branch. For reproducing the results on rgbt234 and 210, you can try to reduce the batch size and learning rate and increase the epoch number when training on lasher.
